index.js

```diff
@@ -0,0 +1,42 @@
+//Benton Green
+module.exports = (function() { 
+
+  function FizzBuzz(input1, input2){
+    this.input1 = input1 || "Fizzy";
+    this.input2 = input2 || "Buzzer";
+  }
+    
+  FizzBuzz.prototype.input = function(startNum, endNum){
+    this.startNum = startNum;
+    this.endNum = endNum + 1; //added one to stopNum so that it would include ending number in list
+    this.finalOutput = writeFizzBuzz(startNum,endNum,this.input1,this.input2);
+  };
+
+  FizzBuzz.prototype.output = function(){
+    return this.finalOutput; 
+  };
+
+  this.writeFizzBuzz = function(startNum, endNum, input1, input2) {
+
+    var outputArray = [];
+
+    for(var x = this.startingNum; x < this.endingNum; x++) {
```

Comment:
`this.startingNum` and `this.endingNum` aren't defined, so this loop never runs. If you change these to `startNum` and `endNum` they work as expected.